spec/system/admin/order_cycles/complex_editing_multiple_product_pages_spec.rb

```diff
@@ -36,9 +36,9 @@
     end
 
     it "select all products" do
-      # replace with scroll_to method when upgrading to Capybara >= 3.13.0
       checkbox_id = "order_cycle_incoming_exchange_0_select_all_variants"
-      page.execute_script("document.getElementById('#{checkbox_id}').scrollIntoView()")
+      elmnt = find_field(id: checkbox_id)
+      scroll_to(elmnt, align: :top)
```

Comment:
- That's great to incorporate the latest recommended method.
- By looking at the flaky spec, we are not getting the element by`checkbox_id`
- Like the proposed changes, I think it's worth trying to use the built-in capybara method as opposed to running the JS script for the purpose
- Let's hope for the best now :)